storage-resize-images/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Version 0.1.9
+
+feature - skip resizing of vector images (Fixes #326). (#329)
+build - replace `mkdirp-promise` with `mkdirp` because `mkdirp-promise` has been deprecated. (#266)
```

Comment:
```suggestion
build - Replaced `mkdirp-promise` with `mkdirp` because `mkdirp-promise` is deprecated. (PR #266)
```

Reply:
I'm not sure if `build` is an accepted release note label?

Reply:
I'm not certain either - should we switch this to 'fixed' to be safe? I'd also buy the argument that this shouldn't be included in the release notes since its not a user-facing change.

Reply:
Yeah, I'd agree... let's change to `fixed`.

I think we should keep the info here in the release notes b/c these notes also a record of what changed in the extension source code just in general.

Reply:
Done!